var/spack/repos/builtin/packages/raja-perf/package.py

```diff
@@ -0,0 +1,325 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import socket
+
+from spack.package import *
+
+from .blt import llnl_link_helpers
+
+
+class RajaPerf(CachedCMakePackage, CudaPackage, ROCmPackage):
+    """RAJA Performance Suite."""
+
+    homepage = "http://software.llnl.gov/RAJAPerf/"
```

Comment:
This is no longer a valid URL.

Reply:
The "closest" non-github URL may be https://software.llnl.gov/#RAJAPerf BUT you still have to search for `RAJAPerf` to narrow it down to the software.

Reply:
It isn't available from https://software.llnl.gov/radiuss/projects/, only `raja` yields the software "box" link.

Reply:
Done !